core/domain/story_domain_test.py

```diff
@@ -454,12 +454,60 @@ def test_is_manager(self):
         self.assertTrue(story_rights.is_manager(self.USER_ID_1))
         self.assertFalse(story_rights.is_manager('fakeuser'))
 
-    def test_story_rights_change_defaults(self):
-        change_dict = story_domain.StoryRightsChange({
-            'cmd': story_domain.CMD_CREATE_NEW
-        }).to_dict()
-        story_rights = story_domain.StoryRightsChange(change_dict)
+
+class StoryRightsChangeTests(test_utils.GenericTestBase):
+    """Test the story rights change domain object."""
+
+    def setUp(self):
+        super(StoryRightsChangeTests, self).setUp()
+        self.STORY_ID = story_services.get_new_story_id()
+        self.story = self.save_new_story(
+            self.STORY_ID, 'user_id', 'Title', 'Description', 'Notes'
+        )
+        self.signup('user@example.com', 'user')
+
+    def test_initializations(self):
```

Comment:
Please take a look at my previous comments carefully. You should have two test methods. One method tests initialization. The other tests to_dict(). Don't lump the two together.

Reply:
Sorry I misunderstood. Changed